configure.py

```diff
@@ -1067,6 +1067,12 @@ def configure_node(o):
   else:
     o['variables']['node_use_dtrace'] = 'false'
 
+  if options.node_use_large_pages or options.node_use_large_pages_script_lld:
+    warn('''Support for mapping to large pages is now a runtime option of
```

Comment:
Maybe to make it clear:
```suggestion
    warn('''The `--use-largepages` and '--use-largepages-script-lld` options have no effect during build time. Support for mapping to large pages is now a runtime option of
```